core/App/ehsm_napi.cpp

```diff
@@ -52,50 +58,53 @@ static char* StringToChar(std::string str)
     }
     return retChar;
 }
+typedef struct {
 
-struct RetJsonObj 
-{
     const int CODE_FAILED = 500;
     const int CODE_SUCCESS = 200;
+
     int code = CODE_SUCCESS;
     std::string msg = "success!";
     std::string jsonStr;
-
-    void addData(std::string key, std::string data)
-    {
+	
+	void setCode(int code){code = code;};
+	void setMessage(string message){msg = message;};
+	void addData(string key, uint32_t data) {
+        if(jsonStr.size() > 0){
+            jsonStr += ",";
+        }
+        jsonStr += "\""+key+"\" : " + "\""+intToString(data)+"\"";
```

Comment:
seem you didn't modify intToString to std::to_string()

Reply:
It will be fix.